### PR TITLE
PU-98

### PR DESCRIPTION
**Pull request by alexkaroleva42**
If you like that solution, I can redo just like that previous task PU-97 with favotiteEvents

### CODE REVIEW

File: controllers/profileController.js [hunk 1]
```diff
@@ -91,99 +91,126 @@ class ProfileController {
...
+    async addNewAttendingEvent(req, res, next) {
         try {
             const { userId } = req.user;
             const eventId = req.params.id;
+            
             const user = await UserInfo.findByPk(userId);
             const event = await EventDetails.findByPk(eventId);
 
             if (!event) {
-                return next(ApiError.badRequest('No such event'));
```
Comment (mastafit) [inline controllers/profileController.js:102]: What was wrong with that? ApiError is specially designed to handle such errors

File: controllers/profileController.js [hunk 2]
```diff
@@ -91,99 +91,126 @@ class ProfileController {
...
-            await user.save();
-            return res.json(user);
+                await UserInfo.update({
+                    attendingEvents: user.attendingEvents
+                }, {
+                    where: { userInfoId: userId }
+                });
 
+            }
+            const updUser = await UserInfo.findByPk(userId);
```
Comment (mastafit) [inline controllers/profileController.js:138]: This part generally looks nice

File: controllers/profileController.js [hunk 3]
```diff
@@ -91,99 +91,126 @@ class ProfileController {
...
         }
     }
 
 
 
-    async deleteOneFavoriteEvent(req, res, next) {
+    async deleteOneAttendingEvent(req, res, next) {
         try {
-            const { userId } = req.user;
+            const {userId} = req.user;
```
Comment (mastafit) [inline controllers/profileController.js:160]: Why delete the spaces?

Comment (alexkaroleva42) [reply controllers/profileController.js:160]: Haven't done autoformatting after

File: controllers/profileController.js [hunk 4]
```diff
@@ -91,99 +91,126 @@ class ProfileController {
...
+            return res.json(user.attendingEvents);
         } catch (error) {
             next(ApiError.internal('Error'));
         }
     }
 
 
 
-    async deleteOneFavoriteEvent(req, res, next) {
+    async deleteOneAttendingEvent(req, res, next) {
```
Comment (mastafit) [inline controllers/profileController.js:158]: The naming is quite strange, are planing on implementing deleteManyAttendingEvents?

File: controllers/profileController.js (hunk 3 above)
Comment (mastafit) [reply controllers/profileController.js:160]: Finish the changes and merge

Comment (mastafit) [reply controllers/profileController.js:160]: Fix that

**Pull request merged**